### ISSUE TITLE
Lock/Unlock State of Icon

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockbox/lockbox-extension#334]
A point of discussion for the future, but I've been thinking about @hmcgaw’s comments regarding the lockbox (locked/unlocked) status indicator, and wanted to pose something to the group... it probably shouldn't be about whether or not your lockbox is authenticated, rather whether or not the current site is secured within lockbox? 

We have the ability in the future to not only adjust lock/unlock based on the type of icon shown, but also a lighter state (a potential preference for logged in vs. logged out) color, and/or numeric indicators (not my preference because of the visual density relative to our real-estate). 

One potential approach could be the following: 
- Color: (dark grey) vs. a lighter shade to represent situations where someone has previously linked FxA but isn't currently authenticated. (This has some challenges for not wanting it to appear disabled...)
- State of Icon (locked vs. unlocked) to represent whether or not there is an entry associated with the current root domain... essentially a locked down icon would signify "we've got something secured in your lockbox associated with this URL.... though this could also be accomplished with color or numbers too... and I'm not sure that the metaphor really makes sense of an open vs. closed lockbox to signify this... 

This takes me to a place where I wonder about any justifiable reasons to have an 'unlocked' looking lockbox... I don't think it makes sense for signed in/signed out unless we move to a world where access is gated from the start via FxA

Instead we handle an "active/inactive" depiction of it for FxA auth use cases, and perhaps color for identifying root domains that match corresponding entries within Lockbox.

**Comment by devinreams** [comment]
>it probably shouldn't be about whether or not your lockbox is authenticated

I'd agree with that. The more I use Lockbox the weirder it is to me to see the unlocked state almost exclusively.

>This takes me to a place where I wonder about any justifiable reasons to have an 'unlocked' looking lockbox... I don't think it makes sense for signed in/signed out unless we move to a world where access is gated from the start via FxA

Do you want to reach a consensus on this before Alpha or Beta? For what it's worth, we can easily remove the "unlocked" icon if that's cleaner and preferred in the short term.

**Comment by changecourse** [comment]
Yeah, I had this on my list to speak with @hmcgaw about, but I think for the short term, it may make sense to revert back to a single icon (locked) for all states in Alpha... then decide on a more formal (and trusted) approach for how we handle this in the Beta.

**Comment by devinreams** [comment]
>I think for the short term, it may make sense to revert back to a single icon (locked) for all states in Alpha

Understood. 👌 

Since I was already in there at #359 I removed the "unlocked" version of the icon. Only "locked" will be shown in all cases for now.

**Comment by jimporter** [comment]
> I'd agree with that. The more I use Lockbox the weirder it is to me to see the unlocked state almost exclusively.

That's probably got a lot to do with the fact that we don't actually have the ability to *lock* the datastore yet.
